tests/store/tracking/test_sqlalchemy_store.py

```diff
@@ -143,6 +145,129 @@ def test_fail_on_multiple_drivers():
         extract_db_type_from_uri("mysql+pymsql+pyodbc://...")
 
 
+@dataclass
+class TestDatabase:
+    db_url: str
+    store: SqlAlchemyStore
+
+
+@pytest.fixture
+def test_database(tmp_path: Path) -> TestDatabase:
+    db_url = _setup_db_uri(tmp_path)
+    store = _get_store(db_url)
+    yield TestDatabase(db_url, store)
+    _cleanup_database(store)
+
+
+def _get_store(db_uri=""):
+    return SqlAlchemyStore(db_uri, ARTIFACT_URI)
```

Comment:
Is it possible to use a temporary directory instead of `ARTIFACT_URI`?

Reply:
Yes, though I don't know the context of why ARTIFACT_URI was provided here rather than tmp dir previously, I think we can simplify here.